types/express-boom/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
We ask that contributors [not disable lint rules](https://github.com/definitelytyped/definitelytyped#lint) for new definitions.